pkgs/shells/antigen/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "2.2.1";
+  name = "antigen";
+
+  src = fetchurl {
+    url = "git.io/antigen";
```

Comment:
Is fetching from `https://github.com/zsh-users/antigen/releases/download/v${version}/antigen.zsh` will be ok?